internal/git/repository.go

```diff
@@ -58,3 +68,21 @@ func (r *Repository) HeadHash() (string, error) {
 
 	return head.Hash().String(), nil
 }
+
+const (
+	defaultBranch string = "master"
```

Comment:
Nit: I really think we should change this to `main` to be in check with common industry practice. I could pull up a lot of references, such as:

- https://github.blog/changelog/2020-10-01-the-default-branch-for-newly-created-repositories-is-now-main/
- https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/

Reply:
AH Ha! I meant to write `main` here.